Source/WebCore/platform/graphics/WidthCache.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2012 Apple Inc. All rights reserved.
+ * Copyright (C) 2012-2023 Apple Inc. All rights reserved.
```

Comment:
Are you an Apple employee? I don't think this is right.

Reply:
Will remove in next commit and add merge-queue. :-)